kafka-ui-react-app/src/components/common/Search/__tests__/Search.spec.tsx

```diff
@@ -11,36 +11,38 @@ jest.mock('use-debounce', () => ({
 describe('Search', () => {
   const handleSearch = jest.fn();
   it('calls handleSearch on input', () => {
-    const component = mount(
-      <ThemeProvider theme={theme}>
-        <Search
-          handleSearch={handleSearch}
-          value=""
-          placeholder="Search bt the Topic name"
-        />
-      </ThemeProvider>
-    );
-    component.find('input').simulate('change', { target: { value: 'test' } });
-    expect(handleSearch).toHaveBeenCalledTimes(1);
-  });
-
-  describe('when placeholder is provided', () => {
-    const component = shallow(
+    render(
       <Search
         handleSearch={handleSearch}
         value=""
         placeholder="Search bt the Topic name"
       />
     );
+    const input = screen.getByPlaceholderText('Search bt the Topic name');
+    userEvent.click(input);
+    userEvent.keyboard('value');
+    expect(handleSearch).toHaveBeenCalledTimes(5);
+  });
+
+  describe('when placeholder is provided', () => {
     it('matches the snapshot', () => {
-      expect(component).toMatchSnapshot();
+      const { baseElement } = render(
+        <Search
+          handleSearch={handleSearch}
+          value=""
+          placeholder="Search bt the Topic name"
+        />
+      );
+      expect(baseElement).toMatchSnapshot();
```

Comment:
you can get ride of snapshot and just check if placeholder is provided and the same if placeholder is not provided